apisix/plugins/request-id.lua

```diff
@@ -51,6 +51,22 @@ local schema = {
                 }
             },
             default = {}
+        },
+        nanoid = {
```

Comment:
why is this block needed when just replacing the dependency would fix the issue?

Reply:
The new dependency provides more controllable parameters and more customized functions, so parameter options are added，just like range_id